src/sentry/static/sentry/app/components/dataExport.tsx

```diff
@@ -9,9 +9,14 @@ import {Organization} from 'app/types';
 import withApi from 'app/utils/withApi';
 import withOrganization from 'app/utils/withOrganization';
 
+//! Coordinate with other ExportQueryType (src/sentry/data_export/base.py)
+export enum ExportQueryType {
+  IssuesByTag = 'Issues-by-Tag',
+  Discover = 'Discover',
+}
```

Comment:
Instead of using the TS: `string | string` thing, I modified it to be an enum which matches one in the backend. If there is some better coordination system please let me know, but for now a comment seems okay.